src/commands/Roll.js

```diff
@@ -4,23 +4,53 @@ class Roll extends Command {
 	constructor(...args) {
 		super(...args);
 
-		this.aliases      = ['roll'];
-		this.module       = 'Fun';
-		this.description  = 'Roll the dice';
-		this.usage        = 'roll [number of dice]';
-		this.example      = 'roll 5';
-		this.cooldown     = 3000;
-		this.expectedArgs = 0;
+		this.aliases 		= ['roll'];
+		this.module 		= 'Fun';
+		this.description 	= 'Roll the dice (support optional size: d4, d8, d10, d12, d20, d00)';
+		this.usage 			= 'roll [size] [number of dice]';
```

Comment:
The spacing isn't right on here

Reply:
This seems to be a github issue actually? It's fine on my editor 🤔